airbyte-ci/connectors/pipelines/README.md

```diff
@@ -122,6 +122,7 @@ Available commands:
 | `--use-remote-secrets`                                         | False    | True                             | If True, connectors configuration will be pulled from Google Secret Manager. Requires the GCP_GSM_CREDENTIALS environment variable to be set with a service account with permission to read GSM secrets. If False the connector configuration will be read from the local connector `secrets` folder. |
 | `--name`                                                       | True     |                                  | Select a specific connector for which the pipeline will run. Can be used multiple time to select multiple connectors. The expected name is the connector technical name. e.g. `source-pokeapi`                                                                                                        |
 | `--support-level`                                              | True     |                                  | Select connectors with a specific support level: `community`, `certified`.  Can be used multiple times to select multiple support levels.                                                                                                                                                             |
+| `--metadata-query`                                              | False     |                                | Filter connectors by metadata query using `simpleeval`. e.g. 'data.ab_internal.ql == 200' |
```

Comment:
Can we add the link to simpleeval? and will everyone need to backslash the quotes in the usage like [here](https://github.com/airbytehq/airbyte/pull/30330/files#r1322259492)?  some more concrete examples of using this in the readme might be helpful

Reply:
especially pointing out that `data` points to the top level of the metadata file - not sure people will know that.